testutil/helpers.go

```diff
@@ -52,9 +52,7 @@ func ConfigureTracing() func(context.Context) {
 		tp = trace.NewTracerProvider(
 			trace.WithSpanProcessor(sp),
 			trace.WithSampler(trace.AlwaysSample()),
-			trace.WithResource(resource.NewSchemaless(
-				attribute.KeyValue{Key: semconv.ServiceNameKey, Value: attribute.StringValue("integration-test-client")},
-			)),
+			trace.WithResource(resource.NewSchemaless(semconv.ServiceName("integration-test-client"))),
```

Comment:
This change is not strictly needed, but while working on this code, I found the `semconv.ServiceName` utility, and thought it was slightly cleaner than to manually construct the `attribute.KeyValue`